alf/environments/suite_highway.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2020 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Suite for loading highway environments.
+    Installation:
+    pip install git+https://github.com/eleurent/highway-env
+"""
+
+import collections
+import gin
+import gym
+import gym.spaces
+
+from alf.environments import suite_gym, alf_wrappers, gym_wrappers, process_environment
+
+try:
+    import highway_env
+except ImportError:
+    highway_env = None
+
+
+def is_available():
+    return highway_env is not None
+
+
+class VectorizeObservation(gym_wrappers.BaseObservationWrapper):
```

Comment:
The class name can be better called `FlattenObservation`; missing docstring for this class

Reply:
Changed class name and added doc string